habitat/tasks/rearrange/utils.py

```diff
@@ -352,6 +349,27 @@ def calc_ik(self, targ_ee: np.ndarray):
         :param targ_ee: 3D target position in the ROBOT BASE coordinate frame
         """
         js = p.calculateInverseKinematics(
-            self.robo_id, self.pb_link_idx, targ_ee, physicsClientId=self.pc_id
+            self.robo_id,
+            self.pb_link_idx,
+            targ_ee,
+            physicsClientId=self.pc_id,
         )
         return js[: self._arm_len]
+
+
+class UsesRobotInterface:
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
I doubt this super().__init__ call is useful.
This whole class does not seem very useful (at least for now), can you add some documentation or a TODO so we know what to do with it in the future?